src/games/strategy/engine/pbem/TripleAWarClubForumPoster.java

```diff
@@ -28,8 +28,6 @@
  */
 public class TripleAWarClubForumPoster extends AbstractForumPoster {
   private static final long serialVersionUID = -4017550807078258152L;
-  private static String m_host = "www.tripleawarclub.org";
```

Comment:
Made this string a static constant: "http://www.tripleawarclub.org/modules/newbb", allowed for additional duplication to be remove.  Note in the diff one can see this string ""http://www.tripleawarclub.org/modules/newbb/reply.php?" twice more already.